wandb/sdk/lib/redirect.py

```diff
@@ -27,6 +29,19 @@
 ANSI_CSI_RE = re.compile("\001?\033\\[((?:\\d|;)*)([a-zA-Z])\002?")
 ANSI_OSC_RE = re.compile("\001?\033\\]([^\a]*)(\a)\002?")
 
+SEP_RE = re.compile(
```

Comment:
Can you add a comment explaining what this is / where you found it?

Reply:
Simplified and comment added.